pkgs/applications/science/biology/slicer3d/default.nix

```diff
@@ -0,0 +1,110 @@
+{
+  stdenv,
+  lib,
+  autoPatchelfHook,
+  makeDesktopItem,
+  glib,
+  fetchzip,
+  pulseaudio,
+  xorg,
+  zlib,
+  fontconfig,
+  libxkbcommon,
+  libGL,
+  libGLU,
+  freetype,
+  cups,
+  unixODBC,
+  libxcrypt-legacy,
+  alsa-lib,
+  postgresql,
+  nss,
+  nspr,
+}:
+
+let
+  pname = "Slicer3D";
+  version = "5.6.2";
+
+  desktopItem = makeDesktopItem {
+    name = "Slicer3D";
+    exec = "SlicerApp-real";
+    desktopName = "Slicer 3D";
+    genericName = "A 3D Slicer Program"; # edit later
+    categories = [ "Graphics" ]; # Edit later
+  };
+in
+stdenv.mkDerivation {
+  inherit pname version;
+  src = fetchzip {
+    name = "Slicer-${version}-linux-amd64.tar.gz";
+    url = "http://download.slicer.org/download?os=linux&stability=release&version=${version}";
+    hash = "sha256-DmJS1yrwJBcAIRnVA8VdsO4u82E1MWX+uvRC+6dEXmM=";
+    extension = "tar.gz";
+  };
+
+  desktopItems = [
```

Comment:
Please move `desktopItems` lower in the file, probably between `installPhase` and `meta` (as it's not a very important attribute, and there's a certain general order that's not codified but expected).